HPXMLtoOpenStudio/resources/geometry.rb

```diff
@@ -569,7 +569,7 @@ def self.apply_occupants(model, runner, hpxml_bldg, num_occ, space, schedules_fi
   end
 
   def self.get_occupancy_default_num(nbeds)
-    return Float(nbeds)
+    return [Float(nbeds), 1.0].max
```

Comment:
My initial reaction is that I don't think this is what we want to do. We shouldn't arbitrarily set to 1 bedroom just to avoid the vacancy code. Wouldn't it be better to change

https://github.com/NREL/OpenStudio-HPXML/blob/0587c5e83857649c8ae225e78ead9e7787af7eab/HPXMLtoOpenStudio/measure.rb#L541-L547

to:

```ruby
    if @hpxml_bldg.building_occupancy.number_of_residents.nil?
      @hpxml_bldg.building_occupancy.number_of_residents = Geometry.get_occupancy_default_num(@nbeds)
    elsif (@hpxml_bldg.building_occupancy.number_of_residents == 0) && (not @apply_ashrae140_assumptions)
      # If zero occupants, ensure end uses of interest are zeroed out
      @hpxml_header.unavailable_periods.add(column_name: 'Vacancy',
```

But I don't know if that will cause problems with any downstream calculations.

Reply:
I implemented this change. Doesn't appear that it caused problems with any downstream calculations.